alfred/ci.py

```diff
@@ -18,6 +18,7 @@ def ci(front, back, e2e, docs):
     if back or no_flags:
         alfred.invoke_command("ci.mypy")
         alfred.invoke_command("ci.ruff")
+        alfred.invoke_command("build.app_provisionning")
```

Comment:
Tests are not running in my IDE. I would advocate adding this step as a fixture in the `tests/conftest.py` module. Execution will be more consistent.

![image](https://github.com/user-attachments/assets/3ea305e6-4905-4d65-90d9-c6ea5526f5e9)


```python
@pytest.fixture(autouse=True)
def build_app_provisionning():
    import os
    import shutil

    root_dir = os.path.dirname(os.path.dirname(__file__))

    if os.path.isdir(os.path.join(root_dir, 'src/writer/app_templates')):
        shutil.rmtree(os.path.join(root_dir, 'src/writer/app_templates'))

    shutil.copytree( os.path.join(root_dir, 'apps'), os.path.join(root_dir, 'src/writer/app_templates'))
```